index.html

```diff
@@ -686,6 +686,51 @@ <h4>Redirected and Proxied External Content</h4>
           </p>
         </section>
       </section>
+
+      <section id="state-tokens">
+        <h3>State Tokens</h3>
+        <blockquote class="informative">
+          Non-normative note: State tokens are similar to ETags, providing a mechanism for clients to track changes
```

Comment:
Perhaps change "track changes in resource state" to "detect a change in resource state" to avoid overstating what one gets from a token